lib/spack/spack/build_systems/meson.py

```diff
@@ -9,7 +9,7 @@
 from typing import List  # novm
 
 from llnl.util.filesystem import working_dir
-from spack.directives import depends_on, variant
+from spack.directives import depends_on, variant, conflicts
```

Comment:
I don't think we need this anymore:
```suggestion
from spack.directives import depends_on, variant
```